test/externalTests.sh

```diff
@@ -73,16 +84,16 @@ function test_truffle
         # Replace fixed-version pragmas in Gnosis (part of Consensys best practice)
         find contracts test -name '*.sol' -type f -print0 | xargs -0 sed -i -e 's/pragma solidity [\^0-9\.]*/pragma solidity >=0.0/'
       fi
-      assertsol="node_modules/truffle/build/Assert.sol"
-      if [ -f "$assertsol" ]
-      then
-        echo "Replace Truffle's Assert.sol with a known good version"
-        rm "$assertsol"
-        wget https://raw.githubusercontent.com/trufflesuite/truffle-core/ef31bcaa15dbd9bd0f6a0070a5c63f271cde2dbc/lib/testing/Assert.sol -o "$assertsol"
```

Comment:
I think this should have stayed if we still support Truffle 4.x

Reply:
They are identical, I checked.

Reply:
Cool! Depends on the truffle version though. We'll see if something breaks in the future anyway.